.gitignore

```diff
@@ -12,3 +12,18 @@ depmap_19Q1_mutation_calls.zip
 DepMap-2019q1-celllines.zip
 gene_effect_corrected.zip
 public_19Q1_gene_cn.zip
+copyNumber_19Q1.rda
```

Comment:
Could you please remove all the rda file from `.gitignore`. These files shouldn't be ignores, they shouldn't be present at all.